examples/dag/dag.yaml

```diff
@@ -34,7 +30,11 @@ nodes:
     name: OpenAI-1
     model: gpt-3.5-turbo
     connection: openai-conn
-    prompt: openai-ai-prompt
+    prompt:
+      id: openai-ai-prompt
```

Comment:
Is `id` required?

Reply:
I don't think we need an `id` in inline prompt.

Reply:
It's not required, we autogenerate in case of a miss